src/Sylius/Bundle/AdminBundle/Resources/assets/entrypoint.js

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
Random question, will bundles for 2.0 be converted to SF dir structure?

Reply:
Yes, but we'll do it as one of the last tasks. Doing it right now will cause many conflicts while upmerging 1.13 -> 2.0.